src/Fabulous.XamarinForms/Views/GestureRecognizers/DropGestureRecognizer.fs

```diff
@@ -0,0 +1,56 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Xamarin.Forms
+
+type IDropGestureRecognizer =
+    inherit Fabulous.XamarinForms.IGestureRecognizer
+
+module DropGestureRecognizer =
+    let WidgetKey =
+        Widgets.register<DropGestureRecognizer> ()
+
+    let AllowDrop =
+        Attributes.defineBindable<bool> DropGestureRecognizer.AllowDropProperty
+
+    let Drop =
+        Attributes.defineEvent<DropEventArgs>
+            "DropGestureRecognizer_Drop"
+            (fun target -> (target :?> DropGestureRecognizer).Drop)
+
+    let DragOver =
+        Attributes.defineEvent<DragEventArgs>
+            "DropGestureRecognizer_DragOver"
+            (fun target -> (target :?> DropGestureRecognizer).DragOver)
+
+    let DragLeave =
+        Attributes.defineEvent<DragEventArgs>
+            "DropGestureRecognizer_DragLeave"
+            (fun target -> (target :?> DropGestureRecognizer).DragLeave)
+
+[<AutoOpen>]
+module DropGestureRecognizerBuilders =
+    type Fabulous.XamarinForms.View with
+        static member inline DropGestureRecognizer<'msg>(onDrop: DropEventArgs -> 'msg) =
+            WidgetBuilder<'msg, IDropGestureRecognizer>(
+                DropGestureRecognizer.WidgetKey,
+                DropGestureRecognizer.Drop.WithValue(fun args -> onDrop args |> box)
+            )
+
+[<Extension>]
+type DropGestureRecognizerModifiers =
+
+    /// <summary>Sets whether users are allowed to drop</summary>
+    /// <param name="value">true to allow users to drop; otherwise, false</param>
+    [<Extension>]
+    static member inline allowDrop(this: WidgetBuilder<'msg, #IDropGestureRecognizer>, value: bool) =
+        this.AddScalar(DropGestureRecognizer.AllowDrop.WithValue(value))
+
+    [<Extension>]
+    static member inline onDragOver(this: WidgetBuilder<'msg, #IDropGestureRecognizer>, onDragOver: 'msg) =
+        this.AddScalar(DropGestureRecognizer.DragOver.WithValue(fun _ -> onDragOver |> box))
```

Comment:
Both `onDragOver` and `onDragLeave` have a useful event args, would be better to take parameter `DragEventArgs -> 'msg`

Reply:
Fixed 😀